src/tpstreams/assets/includes/analytics_devices.html

```
@@ -0,0 +1,502 @@
+          <div class="absolute bottom-0 left-0 w-full py-1 px-1">
+            <div class="h-2 bg-gray-200 dark:bg-neutral-600 rounded-full">
+              <div class="h-full transition-all duration-1000 ease-out rounded-full" :style="'width: ' + (animated ? '12%' : '0%') + '; background-color: #CC0F16;'"></div>
+            </div>
+          </div>
+        </div>
+        <!-- End Card -->
+
+        <!-- Card -->
+        <div class="p-6  bg-gray-100 dark:bg-neutral-700 rounded-lg relative overflow-hidden">
+          <svg class="shrink-0 size-8 mb-4" viewBox="0 0 27600 27600" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+            <linearGradient id="A" gradientUnits="userSpaceOnUse"/>
```

HTMLHint (1.5.0)

[error] The html element name of [ linearGradient ] must be in lowercase.

(tagname-lowercase)